src/components/sendV2/form/form.css

```diff
@@ -142,6 +118,48 @@
   }
 }
 
+.feedbackMessage {
+  margin: 5px 0;
+}
+
+.byteCounter {
+  bottom: 6px;
+  opacity: 0;
+  position: absolute;
+  right: 10px;
+  transition: opacity var(--animation-speed-fast) linear;
+}
+
+.feedback {
+  @mixin contentSmallest;
+
+  box-sizing: border-box;
+  color: var(--color-slate-gray);
+  display: none;
+  justify-content: flex-end;
+  margin-top: 10px;
+  padding: 6px 10px;
+  width: 100%;
+
+  &:global(.error) {
+    color: var(--color-burnt-sienna);
+  }
+
+  &.show {
+    display: flex;
+  }
+
+  &.referenceFeedback {
+    @mixin contentSmallest;
+
+    justify-content: flex-end;
+  }
+}
```

Comment:
If using the feedback component you don't need all the styles, just override the ones needed